docs/osr-resources/playbook.md

```diff
@@ -0,0 +1,89 @@
+---
+id: osr-playbook
+title: OSR Playbook
+sidebar_label: OSR Playbook
+---
+
+This article contains some ideas for solving common problems when starting an OSPO.  It's divided into sections according to the type of problem you are trying to solve.  Eventually, I hope that each item in the list leads to a "recipe" type article to give further clarity on how to proceed with this particular play. 
+
+_**NB:** Work in progress.  Most of this is taken from discussions with Discover on 26/8/22.  Andy Smith (OSPO Lead), Andrew Duckett (Developer Experience) and Andrew Gordon (trying to write Open-Source card-payment gateway testing framework)._
+
+## Funding
+
+_How to make the case for funding an OSPO within an organisation._
+
+- **Legal**: sometimes, OSPOs are started from a position of legal necessity.  Are you using the right licenses?  Does someone have to adjudicate which packages are allowed and which aren't?
+- **Code Duplication**: a lot of times, people need changes to an open-source project.  If you don't allow them to contribute to the original project then they're going to fork the project internally.  If you are a large organisation, you may end up with several different, incompatible forks of some open source project.  This has to be maintained.
```

Comment:
"Code duplication" implies something different from maintaining internal forks. Suggest "Unnecessary code maintenance overhead" or similar.

Reply:
changed to "Maintenance Costs" and linked to a second article which describes the problem in lots of detail